application/src/main/java/org/thingsboard/server/service/ws/telemetry/cmd/v1/RpcCmd.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright © 2016-2023 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.service.ws.telemetry.cmd.v1;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import org.thingsboard.server.service.ws.WsCmd;
+import org.thingsboard.server.service.ws.WsCmdType;
+
+/**
+ * @author lsyer
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class RpcCmd implements WsCmd {
+
+    private int cmdId;  //required
+    private String entityType;  //required
+    private String entityId;  //required
```

Comment:
Instead of these two, just declare `EntityId entityId`

Reply:
Did not understand. An example?